Troubleshooting: add an article about gray boxes in grids (#5131)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/05 DataGrid and TreeList Issues/10 Gray Boxes Appear After Data Loading/05 Use the Latest Version.md

---
##### jQuery

> Update the `devextreme` package to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow the instructions from this topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer: We do not need extra emphasis here. Definiteness in English is a powerful tool. Wield it thoughtfully and responsibly.

From VS In. Where instructions are? They are in the mentioned topic. You do not remove or extract instructions **from** this topic. They will remain exactly where they are.

```suggestion
Update the `devextreme` package to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

> Update the `devextreme` package to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer 2: ```suggestion
Update the `devextreme` package to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to back up the existing project. If you upgrade between major versions, follow the instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

##### Angular

> Update the `devextreme` and `devextreme-angular` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow the instructions from this topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer: > Update **the** `devextreme` and `devextreme-angular` packages

Since we enumerate named entities with AND, we do not need a definite article here according to the new guideline.

> follow **the** instructions from this topic:

Again, I would remove this definite article because leaves another BANG in text for no apparent reason.

In VS from -- where these instructions are? They are in this topic. You do not extract them **from** this topic.

```suggestion
Update `devextreme` and `devextreme-angular` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

> Update `devextreme` and `devextreme-angular` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer 2: ```suggestion
Update `devextreme` and `devextreme-angular` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to back up the existing project. If you upgrade between major versions, follow the instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

##### Vue

> Update the `devextreme` and `devextreme-vue` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow the instructions from this topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer: Same stuff as before because this is exactly the same pattern:

```suggestion
Update `devextreme` and `devextreme-vue` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

> Update `devextreme` and `devextreme-vue` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer 2: ```suggestion
Update `devextreme` and `devextreme-vue` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to back up the existing project. If you upgrade between major versions, follow the instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

##### React

> Update the `devextreme` and `devextreme-react` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow the instructions from this topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer: You know why.

```suggestion
Update `devextreme` and `devextreme-react` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```

> Update `devextreme` and `devextreme-react` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to backup the existing project. If you upgrade between major versions, follow instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).

Reviewer 2: ```suggestion
Update `devextreme` and `devextreme-react` packages to the latest version for quick testing. The issue may be resolved in the most recent update. Do not forget to back up the existing project. If you upgrade between major versions, follow the instructions in the following topic: [Migrate to the New Version](/Documentation/Guide/Common/Migrate_to_the_New_Version/).
```
